tests/sentry/integrations/slack/test_utils.py

```diff
@@ -1,7 +1,8 @@
 import pytest
 import responses
 
-from sentry.integrations.slack.utils import CHANNEL_PREFIX, MEMBER_PREFIX, get_channel_id
+from sentry.integrations.slack.utils import get_channel_id
+from sentry.integrations.slack.utils.channel import CHANNEL_PREFIX, MEMBER_PREFIX
```

Comment:
This is what prompted my comment in `test_integration.py`, I think this is a bit confusing since it isn't clear why the imports aren't coming from the same place when they all concern "channels"